homeassistant/components/vicare/sensor.py

```diff
@@ -572,6 +582,14 @@ class ViCareSensorEntityDescription(SensorEntityDescription, ViCareRequiredKeysM
         value_getter=lambda api: api.getHoursLoadClass5(),
         state_class=SensorStateClass.TOTAL_INCREASING,
     ),
+    ViCareSensorEntityDescription(
+        key="compressor_phase",
+        name="Compressor Phase",
+        icon="mdi:information",
+        value_getter=lambda api: api.getPhase(),
+        entity_category=EntityCategory.DIAGNOSTIC,
+        device_class=SensorDeviceClass.ENUM,
```

Comment:
> The sensor has a limited set of (non-numeric) states. The options property must be set to a list of possible states when using this device class.

I think we should have the `options` defined here as well.

Reply:
only I don't know which values to expect here, so far I have only seen "ready" on my heatpump, others have seen other values https://github.com/somm15/PyViCare/issues/300

Reply:
if we don't know all possible values please remove the device class

```suggestion
```